bin/interpreter.sh

```diff
@@ -42,6 +42,9 @@ while getopts "hp:d:l:v" o; do
             . "${bin}/common.sh"
             getZeppelinVersion
             ;;
+        u)
+            ZEPPELIN_SSH_COMMAND="ssh ${OPTARG}@localhost "
+            ;;
         esac
```

Comment:
@zjffdu yes, I agree, its not as implementation in hadoop, would you recommend something else ?